### DISCUSSION TITLE
Collect by returning multiple separate files

### DISCUSSION THREAD

**Discussion by douglasferreiraufmt** [cjbarrie/academictwitteR#360] [Q&A]
Collect by returning multiple separate files

my collection of 70 thousand tweets returned the result in several files in json format (total of 84). Is this correct? Is it not possible to have a result in a single file? Or would it be possible to merge all these files into one?

**Comment by cjbarrie** [comment]
This is all described on the main [README](https://github.com/cjbarrie/academictwitteR#data-storage) . If you have json files you can bind with `bind_tweets()`

**Reply by douglasferreiraufmt** [reply]
Professor Christopher, thank you for your answer.

I have no experience with programming, I come from another area of knowledge. I'm sorry if my question is very amateurish, but could you please give me an example of how to apply bind_tweets() within a script? I tried to do tests from what is explain in the README, but it didn't work.

Also, is it possible for the collection to return a file in excel, for example, instead of json?

**Comment by douglasferreiraufmt** [comment]
Someone? Please....

**Reply by cjbarrie** [reply]
Please share your code and I’ll have a look at what’s going wrong.

Code describing how to bind tweets is all described [here](https://github.com/cjbarrie/academictwitteR#reformatting)
